### DISCUSSION TITLE
Markodown Docs

### DISCUSSION THREAD

**Discussion by Kukurijek** [fastlane/fastlane#18463] [Ideas]
# Feature Request
It would be nice if Fastlane supported an action to generate Markdown Docs.

**Motivation Behind Feature**
There is already jazzy action for generating docs, but it creates .html file and no .md.
Markdown is quite convenient and you could embed it almost everywhere.

I will try to do it, since I found something already.

**Feature Description**
If you set this action, you will get automatically generated .md files, for Library, SP or whole code. It should be flexible so you can chose access level to include.

**Comment by Kukurijek** [comment]
Has been implemented here  #18464

**Comment by Bob-112** [comment]
As far as I'm concerned the best tool for writing documentation is the one that anyone can actually be bothered to use. Markdown might not be perfect but at least it's simple and commonly understood so there isn't much barrier to writing/updating documentation.


[official site](https://www.targetpayandbenefits.review/)
